sat/flow.py

```diff
@@ -42,7 +44,13 @@ def add_cmllr_transform(
     multiply = net.add_node(
         "signal-matrix-multiplication-f32",
         "apply-cmllr-transform",
-        {"file": os.path.join(str(transform_dir), matrix_name)},
+        {
+            "file": DelayedFormat(
+                "{transform_dir}/{matrix_name}",
+                transform_dir=transform_dir,
+                matrix_name=matrix_name,
+            )
```

Comment:
if `transform_dir` is a Path, then you should be able to use 
```
transform_dir.join_right(matrix_name)
```
to simplify

Reply:
Okay this function is new to me...

Reply:
Okay, changed it and added docstrings (I think we should do this for every code we touch now)